### ISSUE TITLE
ScrCpy made phone screen not open again

### ISSUE THREAD

**Issue by GUser7** [Genymobile/scrcpy#2911]
Hi people,
 first, thanks for this software. Second, I want to report I had encountered a problem, I just discovered this software a few days ago and was wondering if I could turn off the phone screen to save battery life, I found that I just had to press [Ctrl + O] and it did turn off my phone screen. After I was using it I was getting ready to turn on my phone screen again, but the cable suddenly disconnected, the session has stopped, but my phone screen is still off. I was wondering if there is a fix to this, or I have to bring my phone to a store and factory reset it?
 
TLDR; after using the shortcut [CTRL + O] and wanted to turn on my phone screen again, the cable disconnected. Was wondering how to turn the phone screen on again? My screen is still off and functioning, and my morning alarms are killing me.

(Might Help)

-scrcpy version: v1.21 (win 64)

-Samsung A12 (Android 11)
-Asus VivoBook 15 (Win 10)

**Comment by rom1v** [comment]
> -scrcpy version: v1.21 (win 64)

Just to be sure: is the shortcut really <kbd>Ctrl</kbd>+<kbd>o</kbd> for you?

By default, on v1.21, it's MOD+o, where <kbd>MOD</kbd> is <kbd>Alt</kbd> or <kbd>Super</kbd>: https://github.com/Genymobile/scrcpy#shortcuts
Maybe you changed it (`--shortcut-mod`), but it might also suggest that you are not using v1.21, so I just ask for confirmation.

> TLDR; after using the shortcut [CTRL + O] and wanted to turn on my phone screen again, the cable disconnected.

This case should be managed since v1.15 (30714aba34e8b447159beaebeaf4eeab7bd33907).

In any case, pressing physical power button should restore the default power mode (and if the screen was initially on, pressing it once again turn it on).

Do you encounter the same problem if you turn it off with <kbd>MOD</kbd>+<kbd>o</kbd>, then turn it on with <kbd>MOD</kbd>+<kbd>Shift</kbd>+<kbd>o</kbd>?

In old Android versions, there was sometimes a problem with power mode restoration (see 8c8649cfcd710859ce18eab557ed2af8cedb9a42, dcde578a50ffb9d2594c4764a067e9dd328bf55c). But on Android 11, it should not happen.

**Comment by GUser7** [comment]
Hi, thanks for replying. I think I pressed [Alt+O] or [CTRL + ALT + O].

I also tried to press the physical power button of my phone and the screen still does not turn on and yes, I am using v1.21

Also, I can't connect my phone to my pc anymore since I forgot to press 'always allow this device (smt smt)' in the fingerprinting confirmation

I also did not change anything else since I'm kind of new to this stuff.

**Comment by rom1v** [comment]
That's very bad :(

Reboot your device (press <kbd>POWER</kbd> for 10 seconds to force reboot).

**Comment by GUser7** [comment]
Nothing. I tried so many times but nothing :(

**Comment by rom1v** [comment]
The phone does not shutdown if you press <kbd>POWER</kbd> for 10 seconds?

Try 30 seconds: https://support.google.com/android/answer/7664998?hl=en

> hold down the power button for about 30 seconds to restart

**Comment by GUser7** [comment]
Hello, thanks for your help. I just pressed volume down + power button, and it worked. Thanks so much for helping. ILU man (no homo)
